app/src/main/java/io/gnosis/safe/ui/transactions/execution/TxReviewViewModel.kt

```diff
@@ -251,6 +311,14 @@ class TxReviewViewModel
         } ${activeSafe.chain.currency.symbol}"
     }
 
+    fun totalFee(): String? {
```

Comment:
is it possible to add unit tests for the important code areas?

Reply:
of course, I will increase test coverage in a separate pr